src/config.c

```diff
@@ -2543,6 +2551,11 @@ static sds getConfigReplicaOfOption(typeData data) {
     return sdsnew(buf);
 }
 
+int protectedConfigEnabled(int config, client *c) {
```

Comment:
i feel this is a better name?
```suggestion
int allowProtectedAction(int config, client *c) {
```

Reply:
thanks, I didn't like it too